cmd/session-resume.go

```diff
@@ -38,14 +38,17 @@ var sessionResume = cli.Command{
   {{.HelpName}} - {{.Usage}}
 
 USAGE:
-  {{.HelpName}} [SESSION-ID/all]
+  {{.HelpName}} SESSION-ID
+
+SESSION-ID:
```

Comment:
this seems incomplete. Shouldn't this be `$SESSION-ID` or `all`?

Reply:
No resume is not all @krisis